sys/net/gnrc/netif/lorawan/gnrc_netif_lorawan.c

```diff
@@ -89,15 +90,40 @@ static inline void _set_be_addr(gnrc_lorawan_t *mac, uint8_t *be_addr)
 
 void gnrc_lorawan_mcps_indication(gnrc_lorawan_t *mac, mcps_indication_t *ind)
 {
-    (void)mac;
+    gnrc_netif_t *netif = container_of(mac, gnrc_netif_t, lorawan.mac);
+    gnrc_nettype_t nettype = IS_ACTIVE(CONFIG_GNRC_NETIF_LORAWAN_NETIF_HDR)
+                     ? GNRC_NETTYPE_UNDEF
+                     : GNRC_NETTYPE_LORAWAN;
+    uint32_t demux = IS_ACTIVE(CONFIG_GNRC_NETIF_LORAWAN_NETIF_HDR)
```

Comment:
Please assert (port == 0 || port >= 224).In future we may need to accommodate for Mac compliance test scenarios (224) as well.

Reply:
so, basically you add a remote DoS depending of the content of the downlink? (:

Reply:
I can add port 0 though, since that shouldn't go up to the stack

Reply:
As per 1.0.3 spec, 1-224 is the only allowed range of values for the port. We used to enforce it in the application code previously.
Also, in the current scenario, values beyond 255 are also accepted which should return an error, right ?

Reply:
Wait, I just noticed that the LORAMAC constants are already defining this values (at least for reception). So this means, we can rely on that and assume no frame will be sent out of the valid "application specific" range (1-223). However, reception would need to adapt a similar logic.

> Also, in the current scenario, values beyond 255 are also accepted which should return an error, right ?

Considering that the `port` is an 8 bit value, I don't think that would happen

Reply:
done!

Reply:
Great :)